tests/bin/draft/run.t

```diff
@@ -1,78 +1,51 @@
 We need a basic opam project skeleton
 
-    $ cat > CHANGES.md << EOF \
-    > ## 0.1.0\
-    > \
-    > - Some other feature\
-    > \
-    > ## 0.0.0\
-    > \
-    > - Some feature\
-    > EOF
-    $ cat > whatever.opam << EOF \
-    > opam-version: "2.0"\
-    > homepage: "https://github.com/foo/whatever"\
-    > dev-repo: "git+https://github.com/foo/whatever.git"\
-    > description: "whatever"\
-    > EOF
-    $ touch README
-    $ touch LICENSE
-    $ cat > dune-project << EOF \
-    > (lang dune 2.4)\
-    > (name whatever)\
-    > EOF
+  $ cat > CHANGES.md << EOF
+  > ## 0.1.0
+  > 
+  > - Some other feature
+  > 
+  > ## 0.0.0
+  > 
+  > - Some feature
+  > EOF
+  $ cat > whatever.opam << EOF
+  > opam-version: "2.0"
+  > homepage: "https://github.com/foo/whatever"
+  > dev-repo: "git+https://github.com/foo/whatever.git"
+  > description: "whatever"
+  > EOF
+  $ touch README
+  $ touch LICENSE
+  $ cat > dune-project << EOF
+  > (lang dune 2.4)
+  > (name whatever)
+  > EOF
 
 We need to set up a git project for dune-release to work properly
 
-    $ cat > .gitignore << EOF \
-    > _build\
-    > .formatted\
-    > .mdx\
-    > /dune\
-    > run.t\
-    > EOF
-    $ git init 2> /dev/null > /dev/null
-    $ git config user.name "dune-release-test"
-    $ git config user.email "pseudo@pseudo.invalid"
-    $ git add CHANGES.md whatever.opam dune-project README LICENSE .gitignore
-    $ git commit -m "Initial commit" > /dev/null
-    $ dune-release tag -y
-    [-] Extracting tag from first entry in CHANGES.md
-    [-] Using tag "0.1.0"
-    [+] Tagged HEAD with version 0.1.0
+  $ cat > .gitignore << EOF
+  > _build
+  > /dune
+  > run.t
+  > EOF
+  $ git init > /dev/null 2>&1
+  $ git config user.name "dune-release-test"
+  $ git config user.email "pseudo@pseudo.invalid"
+  $ git add CHANGES.md whatever.opam dune-project README LICENSE .gitignore
+  $ git commit -m "Initial commit" > /dev/null
+  $ dune-release tag -y > /dev/null
 
-We do the whole dune-release process
+We do the whole `dune-release` process
 
 (1) distrib
 
-    $ dune-release distrib --dry-run > /dev/null
+  $ dune-release distrib --dry-run > /dev/null
 
 (2) publish
```

Comment:
Might be worth a word about what that should do, i.e. that it should create a draft release on the GH repo, setting draft to true in the API request, writing the release ID in a `.draft_release` file for later consumption by undraft.

Reply:
I'm a bit surprised this test doesn't run `opam submit` in dry-run mode but that's a matter for a different PR!